### DISCUSSION TITLE
Welcome to Deferred Resources Discussions!

### DISCUSSION THREAD

**Discussion by drewhamilton** [Backbase/DeferredResources#43] [General]
## 👋 Welcome!

We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded.

To get started, comment below with an introduction of yourself.

**Comment by Alex009** [comment]
Good to see same ideas as us :) We also separate resource declaration and resolution, but we focused on strings and images from the beginning. Later we shifted this idea to kotlin multiplatform mobile side.
here - https://github.com/icerockdev/moko-resources our library with support android & ios. maybe you see some useful stuff in our lib too

**Reply by drewhamilton** [reply]
Neat! We'd like to support multiplatform someday too (#8) but needed something stable first so started with just Android.
